proposals/required-members.md

```diff
@@ -202,18 +178,22 @@ this proposal should attempt to address it.
 The following 2 attributes are known to the C# compiler and required for this feature to function:
 
 ```cs
-namespace System.Runtime.CompilerServices;
-
-[AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = false)]
-public sealed class RequiredMemberAttribute : Attribute
+namespace System.Runtime.CompilerServices
 {
-    public RequiredMemberAttribute() {}
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = false)]
+    public sealed class RequiredMemberAttribute : Attribute
+    {
+        public RequiredMemberAttribute() {}
+    }
 }
 
-[AttributeUsage(AttributeTargets.Constructor, AllowMultiple = false, Inherited = false)]
-public sealed class SetsRequiredMembersAttribute : Attribute
+namespace System.Diagnostics.CodeAnalysis
```

Comment:
Was this discussed in API review? Wondering if there is a tracking issue or notes for it.

Reply:
Yes. https://github.com/dotnet/runtime/issues/64248. I put it in `System.Diagnostics.CodeAnalysis` because this attribute is intended to be used by user code, while most of the things in `S.R.CS` are extremely rarely used by someone who knows exactly what they're doing.